contrib/babelfishpg_tsql/src/backend_parser/parser.c

```diff
@@ -29,8 +29,6 @@
 #include "src/pltsql.h"
 #include "tcop/tcopprot.h"
 
-int			pgtsql_base_yydebug;
```

Comment:
Would be better to delete this line too : 
https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_4_X_DEV/contrib/babelfishpg_tsql/src/backend_parser/gramparse.h#L90

Reply:
Fixed.